keyrings/codeartifact.py

```diff
@@ -1,18 +1,49 @@
 #!/usr/bin/env python
 
+from functools import cache
+import os
 import re
 import logging
+import configparser
 
 import boto3
-import botocore
```

Comment:
Also discovered another import that doesn't appear to be used.

Reply:
Good catch!